### ISSUE TITLE
Shipping bin, Pam's harvestcraft

### ISSUE THREAD

**Issue by archkazs** [gdarai/SpaceRace#9]
Hey,

My lady was playing  and she found out that since the stacks are maximum for 16 for most of the vegetables, fruits, etc. you cannot trade them for emeralds since the minimum amount is 32.

Cheers,
Archkazs

**Comment by gdarai** [comment]
Oh... That is not good... I will look into the config if I can fix that, thanks.

**Comment by gdarai** [comment]
If you want a quick fix, there is a config file harvestcraft.cfg
I believe that it is enough to change the values in "shiping bin prices"
When this fix will get into some release (0.56 and above) I believe the new setting will be

```"shipping bin prices" {
    I:shippingbinbeesPrice=8
    I:shippingbincropPrice=16
    I:shippingbinfishPrice=8
    I:shippingbinmeatPrice=8
}```
But I must check how it works first... but if you want you can tweak it for now on your end yourself.
